lib/validations.js

```diff
@@ -88,6 +89,17 @@ validations.isAlphanumeric = function isAlphanumeric(paramName, customMessage) {
   );
 }
 
+validations.isArray = function isPlainObject(paramName, customMessage) {
+  return function(params) {
+
+    return {
+      field: paramName,
+      message: customMessage || message(paramName, "must be an array"),
```

Comment:
Most validation helpers use "should" as opposed to "must". So I think it would make sense to change this to 'should'